internal/rundata/rundata_test.go

```diff
@@ -115,6 +117,56 @@ func TestTopology(t *testing.T) {
 }
 
 func TestProperties(t *testing.T) {
+	const (
+		wantUUID        = "TestProperties123"
+		wantPlanID      = "TestProperties"
+		wantDescription = "TestProperties unit test"
+	)
+	TestUUID = ""
+	TestPlanID = ""
+	TestDescription = ""
+
+	metadataText := fmt.Sprintf(`
+uuid: "%s"
+plan_id: "%s"
+description: "%s"
+`, wantUUID, wantPlanID, wantDescription)
+	if err := os.WriteFile("metadata.textproto", []byte(metadataText), 0644); err != nil {
```

Comment:
As discussed yesterday, please only write to t.TempDir() and os.Chdir() there for testing. Avoid clobbering the source tree.

Reply:
Fixed yesterday but forgot to push my commits. Done.